### DISCUSSION TITLE
daisyui.com scroll animation

### DISCUSSION THREAD

**Discussion by PhE** [saadeghi/daisyui#2175] [General]
The daisyui.com front page scroll animation is awesome !
All the daisy ui widgets are functional all the time during the animation transformation (even the theme demo !). Beautiful !

We have some components based on daisy UI we would like to animate.

Is the source code of the front page available ?
If not, can you tell which approach you use for animation ?

**Comment by saadeghi** [comment]
Thank you  
The document site is here  
https://github.com/saadeghi/daisyui/tree/master/src/docs

This is the function:  
https://github.com/saadeghi/daisyui/blob/9b8dc66aeb0d98e18236bfe7b4d926fb8c317390/src/docs/src/routes/%2Bpage.svelte#L44

It returns a number with custom range based on the scroll percentage of a section. Then I use that number for different CSS rules.

**Reply by PhE** [reply]
Thanks a lot !!
